### ISSUE TITLE
HEVC H265 Won't play on Roku Ultra

### ISSUE THREAD

**Issue by zarathustra614** [jasongdove/ErsatzTV#712]
I was running ErsatzTv through Plex on a Roku Express.  All my media is encoded in H265 and all my channels played correctly, minus a few buffering issues.  I switched to a Roku Ultra hoping the better connectivity options would help the buffering, but all the H265 files stopped playing on the Ersatz channels.  The media stills plays if I select it directly through Plex, but not if I try to select it though "Live TV."

I transcoded some files into H264 and they work properly on the Ultra.  I also set up DizqueTV for a test and the H265 files work when selected through Live TV.  The Express also still works.  So it seems like a problem between ErsatzTV and the Ultra.

**Comment by jasongdove** [comment]
Which version of ErsatzTV are you running - docker, windows, etc? Can you screenshot your ffmpeg profile settings? Do you see any errors in the log files when the files fail to play?

**Comment by zarathustra614** [comment]
I'm running the Windows version of ErsatzTV, v0.4.4

I attached my transcode settings below.  I've typically been running with no transcodes so I don't stress my system and it seems to work fine on other clients.  I also tried using the default settings as well on the Ultra.

I just tried to start a movie around 8:15 and it wouldn't play.  I took a screenshot of the logs.

![No Transcode](https://user-images.githubusercontent.com/102309902/160032126-071887a3-51c2-429c-993b-34c5652795bb.png)
![Default Transcode](https://user-images.githubusercontent.com/102309902/160032127-4aa4373a-4713-41f0-b3e5-71f524ed965f.png)
![ErsatzTv Error Log](https://user-images.githubusercontent.com/102309902/160032129-6466c2fc-5454-4eb3-9cff-e10ccf91e106.png)

**Comment by jasongdove** [comment]
The `Transcode`, `Normalize Video` and `Normalize Audio` checkboxes will likely be removed from an upcoming release and will always act as if they are enabled. For the two primary streaming modes, `HLS Segmenter` and `MPEG-TS`, transcoding and normalization is always required for continuous playback.

`HLS Direct` is the exception here, in that it copies video and audio streams, but it is only known to work with TiviMate on Android.

Since Plex requires `MPEG-TS` mode, it also requires transcoding, video normalization and audio normalization.

The reality is that most users will require some form of hardware acceleration for smooth playback. The requirements for normalizing content for an IPTV stream are much higher than e.g. direct playing VOD content with a dedicated client.

Do you have an Intel iGPU available, or an NVIDIA GPU?

**Comment by zarathustra614** [comment]
I checked off the three boxes you mention above and re-enabled transcoding.  Everything seems to be running smoothly with both H264 and H265 media.  The initial CPU usage spike I was worried about also goes away after a couple of minutes so I'm not worried about transcoding overtaxing my server.

I have a NVIDIA 1660 GPU.  I don't have hardware acceleration enabled but I haven't noticed any problems with playback since I changed the settings around.
